client/app/(auth)/components/loginForm.tsx

```diff
@@ -0,0 +1,105 @@
+'use client'
+
+import Image from 'next/image'
+// import { FcGoogle } from 'react-icons/fc'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  RouterLink,
+  Input,
+  Button,
+} from '@/_components/ui'
+import { useLogin } from '../hooks'
+
+const LoginForm = () => {
+  const { form, onSubmit, isPending } = useLogin()
+
+  return (
+    <div className='flex w-full flex-col items-center bg-amber-50 md:w-[680px] md:p-10'>
+      <Image
+        src='/images/logo/logo.png'
+        width={200}
+        height={40}
+        alt='Fairy Share logo'
+      />
```

Comment:
Sorry, I didn't notice this warning. I will fix it in the upcoming register PR.
<img width="511" alt="Screenshot 2024-03-07 at 2 40 35 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/26a082d8-3f54-4b11-9668-02fe16c9ffbf">

Reply:
I was able to fix it by applying the exact ratio of the original image. 
Let me know if you know a better way to avoid the warn!